CONTRIBUTING.rst

```diff
@@ -0,0 +1,103 @@
+============
+Contributing
+============
+
+Contributions are welcome, and they are greatly appreciated! Every little bit helps, and credit will always be given.
+
+You can contribute in many ways:
+
+Types of Contributions
+----------------------
+
+Report Bugs
+~~~~~~~~~~~
+
+Report bugs at https://github.com/shopkeep/shpkpr/issues.
+
+If you are reporting a bug, please include:
+
+* Your operating system name and version.
+* Did you install with ``pip`` or ``docker``?
+* Are you using a released version or a ``git`` checkout?
+* Any details about your local setup that might be helpful in troubleshooting.
+* Detailed steps to reproduce the bug.
+
+Fix Bugs
+~~~~~~~~
+
+Look through the GitHub issues for bugs. Anything tagged with "bug" is open to whoever wants to implement it.
+
+Implement Features
+~~~~~~~~~~~~~~~~~~
+
+Look through the GitHub issues for features. Anything tagged with "feature" is open to whoever wants to implement it.
+
+Write Documentation
+~~~~~~~~~~~~~~~~~~~
+
+shpkpr could always use more documentation, whether as part of the official shpkpr docs, in docstrings, or even on the web in blog posts, articles, and such.
+
+Submit Feedback
+~~~~~~~~~~~~~~~
+
+The best way to send feedback is to file an issue at https://github.com/shopkeep/shpkpr/issues.
+
+If you are proposing a feature:
+
+* Explain in detail how it would work.
+* Keep the scope as narrow as possible, to make it easier to implement.
+* Remember that contributions are welcome :)
+
+Get Started!
+------------
+
+Ready to contribute? Here's how to set up ``shpkpr`` for local development.
+
+1. Fork the ``shpkpr`` repo on GitHub.
+
+2. Clone your fork locally::
+
+    $ git clone git@github.com:your_name_here/shpkpr.git
+
+3. Install your local copy into a virtualenv. Assuming you have virtualenvwrapper installed, this is how you set up your fork for local development::
```

Comment:
I think it'd be nice if this linked to [virtualenvwrapper](https://virtualenvwrapper.readthedocs.org/en/latest/).